build.sh

```diff
@@ -646,7 +646,8 @@ function run() {
     echo "Running frontend apps..."
     run_frontend
 
-    # Start backend with initial output but without final output/wait
+    # Start backend with security disabled to set up
+    export THUNDER_SKIP_SECURITY=true
```

Comment:
Shall we update powershell script as well?

Reply:
updated